reference/interbit-ui-components/CONTRIBUTING.adoc

```diff
@@ -0,0 +1,205 @@
+= Contributing
+
+This document is for developers who wish to contribute to the
+`interbit-ui-components` package within the `interbit` repo. For general
+information on contributing to
+link:https://github.com/interbit/interbit[Interbit], please see this
+link:https://github.com/interbit/interbit/blob/master/CONTRIBUTING.md[Contribution Guide].
+
+
+== Creating components
+
+=== File structure
+
+[cols="1a,1a", options="header"]
+|===
+| Directory or file
+| Description
+
+
+| `src/assets`
+| Location for image assets.
+
+| `src/components`
+| React components exported from this package.
+
+| `src/components/BlockExplorer`
+| An interactive component displaying a chain's state, its blocks, and actions
+dispatched.
```

Comment:
This doesn't read quite right. Replace 'displaying' with 'for displaying' or 'that displays'.